src/DataViews/DataViewTest.cpp

```diff
@@ -22,3 +29,28 @@ TEST(DataView, FormatValueForCsvEscapesQuotesInString) {
   constexpr std::string_view kExpectedResult("\"string\"\"with\"\"quotes\"");
   EXPECT_EQ(kExpectedResult, FormatValueForCsv(kInput));
 }
+
+constexpr uint64_t kValuesNum = 5;
+const std::array<std::string, kValuesNum> kValues = {"a", "b", "c", "d", "e"};
+const std::array<std::string, kValuesNum> kCSVFormattedValues = [] {
```

Comment:
Nit: `kCsvFormattedValues`.

Also, if these constants are only used in one test, i.e., `WriteLineToCsvIsCorrect`, I would like to define them inside the test;  if these constants are shared by several tests, then put them in the anonymous namespace.